### PR TITLE
feat: store coverage data in memory-db

### PR DESCRIPTION
**Pull request by zrock-dev**
## Related Tickets & US
- [CS-43-11 Store coverage data in DB](https://app.clickup.com/t/868b2k25u)

## Description

Implement a JSON mapper to business logic entities.

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Implemented unit tests

```
$ ./gradlew test
```

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

- - -
This change is [Reviewable](https://reviewable.io/reviews/SoftDevSix/ArgosAPI/23)

### CODE REVIEW

File: src/main/java/com/softdevsix/api/repositories/IProjectRepository.java [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.UUID;
 
 @Repository
 public interface IProjectRepository {
     Project findById(UUID id);
     void save(Project project);
+
```
Comment (salet-gutierrez-jalafund) [inline src/main/java/com/softdevsix/api/repositories/IProjectRepository.java:14]: delete this line

**Review by salet-gutierrez-jalafund — APPROVED** [review]
Good job

**Review by GabyGV-GGV — APPROVED** [review]
Good job!

**Comment by zrock-dev** [comment]
This PR should have been merged to develop.

**Pull request merged**